src/table/components/__tests__/PaginationContent.spec.jsx

```diff
@@ -0,0 +1,246 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import PaginationContent from '../PaginationContent';
+import * as handleAccessibility from '../../utils/handle-accessibility';
+
+describe('<PaginationContent />', () => {
```

Comment:
These are a combination of the old test for the TablePaginationActions and some from TableWrapper